ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -2290,3 +2290,52 @@ def test_paddle_trunc(
         on_device=on_device,
         x=x[0],
     )
+
+
+@st.composite
+def _draw_paddle_diagonal(draw):
+    _dtype, _x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            min_num_dims=2,
+            max_num_dims=2,
```

Comment:
could you kindly increase this number since `paddle.diagonal` can take higher dimensional inputs as well [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/diagonal_en.html#:~:text=x%20(Tensor)%20%E2%80%93%20The%20input%20tensor%20x.%20Must%20be%20at%20least%202%2Ddimensional.%20The%20input%20data%20type%20should%20be%20bool%2C%20int32%2C%20int64%2C%20float16%2C%20float32%2C%20float64.)